pkgs/development/compilers/llvm/16/llvm/default.nix

```diff
@@ -383,7 +385,10 @@ in
   checkTarget = "check-all";
 
   # For the update script:
-  passthru.monorepoSrc = monorepoSrc;
+  # passthru.monorepoSrc = monorepoSrc;
+  passthru = {
+    inherit monorepoSrc enableZstd;
+  };
```

Comment:
```suggestion
  passthru = { inherit enableZstd; };
```
(see above)

Reply:
So it's unnecessary to set `passthru.monorepoSrc` here?